packages/documentation-site/patternfly-docs/content/design-guidelines/components/pagination/pagination.md

```diff
@@ -6,65 +6,86 @@ section: components
 import '../components.css';
 
 ## Usage
-Add pagination above and below content views so that users can easily navigate as they read through listed items on a page.
+Pagination allows users to conveniently navigate through content-dense components or page elements.
 
 <div class="ws-docs-content-img">
 ![a full page demo with a table and a pagination component within a toolbar.](./img/pagination-elements.svg)
 </div>
 
-1. **Top pagination:** Always right-align pagination in the top toolbar, above content views.
+1. **Top pagination:** Placed above the component it controls and aligned to the right.
+
+2. **Bottom pagination:** Placed below the component it controls. Right-aligned for desktop views and center-aligned for mobile. If you want bottom pagination to be sticky so that it remains in view as users scroll through long content, place it directly after the component it controls and refer to the [React sticky pagination](/components/pagination#sticky) or [HTML sticky pagination](/components/pagination/html#bottom-sticky) examples for implementation guidance.
 
-2. **Bottom pagination:** Right-align pagination below content views, too. If you want the bottom pagination to be sticky so it does not scroll off the page when the content is taller that the viewport, place the pagination in a footer page section and apply the `pf-m-sticky-bottom` modifier section to 
-create a sticky page footer.
 ## Variations 
+
+There are 3 pagination variations you can choose from, depending on your use case:
+- [Full pagination:](#full-pagination) The default component to use for most desktop views when space is not a major constraint. 
+- [Compact pagination:](#compact-pagination) Used for narrow spaces or as the primary control for mobile views.
+- [Indeterminate pagination:](#indeterminate-pagination) Used when the total number of items is unknown.
+
 ### Full pagination
 
-Use full pagination unless you're restricted to a narrow workspace, then use <a href="#compact-pagination">compact pagination</a>. Full pagination occupies more toolbar territory, so you'll have limited room for other items like bulk selectors, buttons, filters, or input fields.
+<div class="ws-docs-content-img">
+![Annotations for each element of a full pagination.](./img/full-pagination.svg)
```

Comment:
I think this image isn't right? It says "1-10 of 50" (implies you're on page 1 of 5 pages), but the number input says "10 of 50" - it should say "1 of 5". If you edit one of the react pagination demos, it will show you what it should look like

<img width="958" height="594" alt="Image" src="https://github.com/user-attachments/assets/acd4518e-641b-4874-87f7-fe5f97d1b56a" />